korman/exporter/convert.py

```diff
@@ -522,6 +559,11 @@ def _(temporary, parent):
             return temporary
 
         def do_pre_export(bo):
+            if bo.type == "ARMATURE":
```

Comment:
This feels a little out of place, but I haven't wrapped my head around what's going on well enough to say what should change.